go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/CosmWasm/wasmvm v1.0.0-soon2.0.20211209162446-b96c92d21aaf // indirect
```

Comment:
should be `github.com/CosmWasm/wasmvm v1.0.0-beta3` ?

Reply:
nope, it is go doing its tag breaking thing.  but let me check the commit height to be certain.

Reply:
ah.  

More fixing needed.

Reply:
oop.  That would have broken everything...